scripts/configs.js

```diff
@@ -33,22 +32,61 @@ export const calcEnvironment = () => {
   return environment;
 };
 
-function buildConfigURL(environment, root = '/') {
+function buildConfigURL(environment) {
   const env = environment || calcEnvironment();
   let fileName = 'configs.json';
   if (env !== 'prod') {
     fileName = `configs-${env}.json`;
   }
-  const configURL = new URL(`${window.location.origin}${root}${fileName}`);
+  const configURL = new URL(`${window.location.origin}/${fileName}`);
   return configURL;
 }
 
+function applyMetadataOverridesToConfig(config) {
+  const website = getMetadata('commerce-website');
```

Comment:
Can you make this code more generic? For example to easily add more parameters if needed. e.g. if you have customers that use dedicated Commerce environments incl. CS environment IDs for individual stores. For example a lot of customer split their US, EMEA and APAC business.

Reply:
Done!

Reply:
How did you resolve this Carlos? And what are you suggesting @herzog31 ? To add other overrides to metadata like commerce endpoint?

Reply:
```
 const updates = new Map([
    ['commerce.headers.cs.Magento-Website-Code', website],
    ['commerce.headers.cs.Magento-Store-Code', store],
    ['commerce.headers.cs.Magento-Store-View-Code', storeview],
    ['commerce.headers.all.Store', storeview],
  ]);
```